src/routes/swagger-ui.ts

```diff
@@ -52,11 +52,11 @@ function getSwaggerHtml() {
           urls: [
             {
               url: "/spec",
-              name: "TSOA Spec"
+              name: "Oauth api"
```

Comment:
```suggestion
              name: "OAuth api"
```

Seeing as we're being told off for our English :smile:  This is how they case it

Probably should also be API but I :heart:  following the casing that the project uses